### PR TITLE
exec 2024 

### PR DESCRIPTION
**Pull request by fattyhope**
Maybe I need to create a separate branch to only update the exec of 2024 since I have random commits from last year, see below: 

- Updated font version 
- I wanted to take down the sponsorship page since it wasn't being used and we don't take payments from stripe anymore (iirc) 

Here's a snapshot of the component I want to merge to main. 

image

Let me know if there are any typos and/or changes wanted to be made. 

Thanks, 
Jackie

### CODE REVIEW

**Comment by bradleysigma** [comment]
Hmmmm, I prefer #322.

File: content/about/_index.html [hunk 1]
```diff
@@ -121,27 +121,27 @@ <h1 class="title is-2">The Executive</h1>
...
                         image-class="committee-headshot"
                         class="committee-element box"
                     >}}
                     {{< committee-photo 
-                        src="/img/media/exec/sonja.jpg" 
-                        title="Sonja McNeilly"
-						subtitle="(@Martini9)"
+                        src="/img/media/exec/richie.jpg" 
+                        title="Richard"
+						subtitle="(@Jacron)"
```
Comment (jenseni-git) [inline content/about/_index.html:144]:
```suggestion
						subtitle="(@jacron0085)"
```

File: content/about/_index.html [hunk 2]
```diff
@@ -150,78 +150,77 @@ <h1 class="title is-2">The Executive</h1>
...
+                </div>
+                <div class="committee-headshots">
                     {{< committee-photo 
-                        src="/img/media/exec/miguel.jpg" 
-                        title="Miguel Valencia"
-                        subtitle="(@Miggy)"
-                        caption="Design Officer"
+                        src="/img/media/exec/ami.jpg" 
+                        title="Amy"
+                        subtitle="(@firefiredfries)"
```
Comment (jenseni-git) [inline content/about/_index.html:181]:
```suggestion
                        subtitle="(@firefriedfries)"
```

File: content/about/_index.html [hunk 3]
```diff
@@ -150,78 +150,77 @@ <h1 class="title is-2">The Executive</h1>
...
                         image-class="committee-headshot"
                         class="committee-element box"
                     >}}
                     {{< committee-photo 
-                        src="/img/media/exec/isaac.jpg" 
-                        title="Isaac Beh"
-                        subtitle="(@indium)"
+                        src="/img/media/exec/rohan.jpg" 
+                        title="Rohan"
+                        subtitle="(@mainbreaker)"
```
Comment (jenseni-git) [inline content/about/_index.html:216]:
```suggestion
                        subtitle="(@.mainbreaker)"
```

File: content/about/_index.html [hunk 4]
```diff
@@ -150,78 +150,77 @@ <h1 class="title is-2">The Executive</h1>
...
                         image-class="committee-headshot"
                         class="committee-element box"
                     >}}
                     {{< committee-photo 
-                        src="/img/media/exec/nazeef.jpg" 
-                        title="Nazeef Hamid"
-                        subtitle="(@nazzle)"
+                        src="/img/media/exec/alex.jpg" 
+                        title="Alex"
+                        subtitle="(@cheerfulmos)"
```
Comment (jenseni-git) [inline content/about/_index.html:223]:
```suggestion
                        subtitle="(@cheerfulmoss)"
```

File: layouts/partials/head.html [hunk 5]
```diff
@@ -10,7 +10,7 @@
 <!-- <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.1/css/bulma.min.css"> -->
 {{ "<!-- Custom CSS -->" | safeHTML }}
 {{ "<!-- Font Awesome -->" | safeHTML }}
-<script defer src="https://use.fontawesome.com/releases/v6.1.1/js/all.js"></script>
+<script defer src="https://use.fontawesome.com/releases/v6.5.1/js/all.js"></script>
```
Comment (jenseni-git) [inline layouts/partials/head.html:13]:
```suggestion
<script defer src="https://use.fontawesome.com/releases/v6.1.1/js/all.js"></script>
```

**Review by jenseni-git — CHANGES_REQUESTED** [review]
/Suggestions change discord tags.

I'd personally rather have full names, but aight chief 🫡.

Also, did you tinker with fontawesome at all? If not, probably doesn't need to change.

othw, :lgtm:

File: layouts/partials/head.html (hunk 5 above)
Comment (JamesDearlove) [reply layouts/partials/head.html:13]: I'm actually happy for the Font Awesome version to be bumped, no reason it can't be

**Review by JamesDearlove — COMMENTED** [review]
Personally (and due to precedent), I'd also prefer full names on the about page. Otherwise after Discord tag changes I think it's all good.

Note for mergers: Make sure you squash and merge this commit, as Jackie has reverted in this PR. Also squashes look neater (imo)

File: content/about/_index.html [hunk 6]
```diff
@@ -121,27 +121,27 @@ <h1 class="title is-2">The Executive</h1>
             <div class="column is-8">
                 <div class="committee-headshots">
                     {{< committee-photo 
-                        src="/img/media/exec/rachel.jpg" 
-                        title="Rachel Chiong"
-                        subtitle="(@rachel)"
+                        src="/img/media/exec/iain.jpg" 
+                        title="Iain"
```
Comment (jenseni-git) [inline content/about/_index.html:125]:
```suggestion
                        title="Iain Jensen"
```

File: content/about/_index.html [hunk 7]
```diff
@@ -121,27 +121,27 @@ <h1 class="title is-2">The Executive</h1>
...
                         link="mailto:president@uqcs.org"
                         image-class="committee-headshot"
                         class="committee-element box"
                     >}}
                     {{< committee-photo 
-                        src="/img/media/exec/limao.jpg" 
-                        title="Limao Chang"
-                        subtitle="(@lmao)"
+                        src="/img/media/exec/jacki.jpg" 
+                        title="Jackie"
```
Comment (jenseni-git) [inline content/about/_index.html:134]:
```suggestion
                        title="Jackie Dinh Chang"
```

File: content/about/_index.html [hunk 8]
```diff
@@ -121,27 +121,27 @@ <h1 class="title is-2">The Executive</h1>
...
                         link="mailto:secretary@uqcs.org"
                         image-class="committee-headshot"
                         class="committee-element box"
                     >}}
                     {{< committee-photo 
-                        src="/img/media/exec/sonja.jpg" 
-                        title="Sonja McNeilly"
-						subtitle="(@Martini9)"
+                        src="/img/media/exec/richie.jpg" 
+                        title="Richard"
```
Comment (jenseni-git) [inline content/about/_index.html:143]:
```suggestion
                        title="Richard Yu"
```

File: content/about/_index.html [hunk 9]
```diff
@@ -150,78 +150,77 @@ <h1 class="title is-2">The Executive</h1>
                 </div>
                 <div class="committee-headshots">
                     {{< committee-photo 
-                        src="/img/media/exec/faith.jpg" 
-                        title="Faith McKay"
-                        subtitle="(@faith)"
+                        src="/img/media/exec/bahareh.jpg" 
+                        title="Bahareh"
```
Comment (jenseni-git) [inline content/about/_index.html:154]:
```suggestion
                        title="Bahareh Khiabani"
```

File: content/about/_index.html [hunk 10]
```diff
@@ -150,78 +150,77 @@ <h1 class="title is-2">The Executive</h1>
...
                         caption="Media Officer"
                         image-class="committee-headshot"
                         class="committee-element box"
                     >}}
                     {{< committee-photo 
-                        src="/img/media/exec/jackie.jpg" 
-                        title="Jackie Chang"
-                        subtitle="(@Ja)"
+                        src="/img/media/exec/jesse.jpg" 
+                        title="Jesse"
```
Comment (jenseni-git) [inline content/about/_index.html:162]:
```suggestion
                        title="Jesse Wang"
```

File: content/about/_index.html [hunk 11]
```diff
@@ -150,78 +150,77 @@ <h1 class="title is-2">The Executive</h1>
...
                         caption="Events Officer"
                         image-class="committee-headshot"
                         class="committee-element box"
                     >}}
                     {{< committee-photo 
-                        src="/img/media/exec/kwan.jpg" 
-                        title="Kwan Li"
-                        subtitle="(@Backpack)"
+                        src="/img/media/exec/harsha.jpg" 
+                        title="Harsha"
```
Comment (jenseni-git) [inline content/about/_index.html:170]:
```suggestion
                        title="Harsha Varma"
```

File: content/about/_index.html [hunk 12]
```diff
@@ -150,78 +150,77 @@ <h1 class="title is-2">The Executive</h1>
...
                     >}}
+                </div>
+                <div class="committee-headshots">
                     {{< committee-photo 
-                        src="/img/media/exec/miguel.jpg" 
-                        title="Miguel Valencia"
-                        subtitle="(@Miggy)"
-                        caption="Design Officer"
+                        src="/img/media/exec/ami.jpg" 
+                        title="Amy"
```
Comment (jenseni-git) [inline content/about/_index.html:180]:
```suggestion
                        title="Amy Hu"
```

File: content/about/_index.html [hunk 13]
```diff
@@ -150,78 +150,77 @@ <h1 class="title is-2">The Executive</h1>
...
                         class="committee-element box"
                     >}}
-                </div>
-                <div class="committee-headshots">
                     {{< committee-photo 
-                        src="/img/media/exec/campbell.png" 
-                        title="Campbell McFadden"
-                        subtitle="(@campbell)"
+                        src="/img/media/exec/anshul.jpg" 
+                        title="Anshul"
```
Comment (jenseni-git) [inline content/about/_index.html:187]:
```suggestion
                        title="Anshul Dadhwal"
```

File: content/about/_index.html [hunk 14]
```diff
@@ -150,78 +150,77 @@ <h1 class="title is-2">The Executive</h1>
...
+                        subtitle="(@anshul.cpp)"
                         image-class="committee-headshot"
                         class="committee-element box"
                     >}}
                     {{< committee-photo 
-                        src="/img/media/exec/connor.jpg" 
-                        title="Connor Geissmann"
-                        subtitle="(@Geisso)"
+                        src="/img/media/exec/tom.jpg" 
+                        title="Tom"
```
Comment (jenseni-git) [inline content/about/_index.html:194]:
```suggestion
                        title="Tom Cameron"
```

File: content/about/_index.html [hunk 15]
```diff
@@ -150,78 +150,77 @@ <h1 class="title is-2">The Executive</h1>
...
+                        subtitle="(@moboso)"
                         image-class="committee-headshot"
                         class="committee-element box"
                     >}}
                     {{< committee-photo 
-                        src="/img/media/exec/elias.jpg" 
-                        title="Elias Blanch"
-                        subtitle="(@Tired Camp Counsellor)"
+                        src="/img/media/exec/roxie.jpg" 
+                        title="Roxie"
```
Comment (jenseni-git) [inline content/about/_index.html:201]:
```suggestion
                        title="Roxie Cunningham"
```

File: content/about/_index.html [hunk 16]
```diff
@@ -150,78 +150,77 @@ <h1 class="title is-2">The Executive</h1>
...
+                        subtitle="(@._hi____.)"
                         image-class="committee-headshot"
                         class="committee-element box"
                     >}}
                     {{< committee-photo 
-                        src="/img/media/exec/iain.jpg" 
-                        title="Iain Jensen"
-                        subtitle="(@I Am The Lain)"
+                        src="/img/media/exec/thommy.jpg" 
+                        title="Thomas"
```
Comment (jenseni-git) [inline content/about/_index.html:208]:
```suggestion
                        title="Thomas Horsey"
```

File: content/about/_index.html [hunk 17]
```diff
@@ -150,78 +150,77 @@ <h1 class="title is-2">The Executive</h1>
...
+                        subtitle="(@thomashorsey)"
                         image-class="committee-headshot"
                         class="committee-element box"
                     >}}
                     {{< committee-photo 
-                        src="/img/media/exec/isaac.jpg" 
-                        title="Isaac Beh"
-                        subtitle="(@indium)"
+                        src="/img/media/exec/rohan.jpg" 
+                        title="Rohan"
```
Comment (jenseni-git) [inline content/about/_index.html:215]:
```suggestion
                        title="Rohan Kollambalath"
```

File: content/about/_index.html [hunk 18]
```diff
@@ -150,78 +150,77 @@ <h1 class="title is-2">The Executive</h1>
...
+                        subtitle="(@mainbreaker)"
                         image-class="committee-headshot"
                         class="committee-element box"
                     >}}
                     {{< committee-photo 
-                        src="/img/media/exec/nazeef.jpg" 
-                        title="Nazeef Hamid"
-                        subtitle="(@nazzle)"
+                        src="/img/media/exec/alex.jpg" 
+                        title="Alex"
```
Comment (jenseni-git) [inline content/about/_index.html:222]:
```suggestion
                        title="Alex Burow"
```

**Review by jenseni-git — APPROVED** [review]
Add last names to names

**Comment by jenseni-git** [comment]
Yay woo finished.

**Pull request merged**